src/Service.Tests/Unittests/RestServiceUnitTests.cs

```diff
@@ -70,7 +70,6 @@ public class RestServiceUnitTests
         /// <param name="route">The route to be parsed.</param>
         /// <param name="path">An invalid path for the given route.</param>
         [DataTestMethod]
-        [DataRow("/foo/bar", "foo")]
```

Comment:
I would argue that this test case is correct to have in "ErrorForInvalidRouteAndPathToParseTest" because it implies that the URL entered into the browser or api client was `https://localhost:5000//api/entity`.

`GetRouteAfterPathBase()` requires a URL with no starting slashes and the route value calculated by the RestController in `        public async Task<IActionResult> Find(string route)` will remove the first slash  leaving `/api/entity` as the route.

Reply:
added back